tools/functional/api/test_merge_requests.py

```diff
@@ -29,6 +29,16 @@ def test_merge_requests(project):
     )
 
 
+def test_merge_request_get(project):
+    mr = project.mergerequests.list()[0]
+    mr_iid = mr.iid
+    mr = project.mergerequests.get(mr_iid)
+    assert mr.iid == mr_iid
```

Comment:
that's not really part of this test. That should be tested somewhere else already. I think we can safely get rid of that first assert

Reply:
In the functional tests I don't see any tests for mergerequests.get(), except in the EE part there happens to be a get.

Reply:
Mhmm okay, maybe then we keep it in a separate test.

Reply:
> Mhmm okay, maybe then we keep it in a separate test.

Sorry, I don't quite understand what you would like me to do. Happy to change things but I'm a bit confused.

Reply:
Could this make use of the new merge request fixture that you added @JohnVillalovos, that might clean it up a bit? https://github.com/python-gitlab/python-gitlab/blob/master/tests/functional/conftest.py#L232